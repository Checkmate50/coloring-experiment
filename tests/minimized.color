x : int;
y : int;
z : float;
w : float;

x < y;
y < z;
w < z;

int;
float;
???;