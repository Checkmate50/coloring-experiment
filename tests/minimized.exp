x : int;
y : int;
z : float;
w : float;
q : T;

x < y;
y < z;
w < z;

int;
float;
???;
T;