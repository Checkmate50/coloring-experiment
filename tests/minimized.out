x;
w;
y;
z;
q;
